### DISCUSSION TITLE
Missing information on build process

### DISCUSSION THREAD

**Discussion by samueldrai** [adonisjs/core#4624] [Help] [answered]
Hi!
I have built an app using the default `web` starter-kit. 
However, when I build my app following the instructions on the official docs, I get a successful build but the `app.js` and `app.css` can't be resolved. 

Is there something to be configured on the `vite.config.ts` and/or `config/vite.ts` to bundle the `app.js` and `app.css` files? I would like those files to be returned by my Node server, not a CDN. 

```sh
Failed to load url resources/js/app.js (resolved id: resources/js/app.js). Does the file exist?
```

To reproduce the bug, just create a new Adonis app using the official `web` starter kit and run 

```sh
pnpm build
cd build
pnpm i --prod
node bin/server.js
```

Thanks a lot for your help!

**Comment by RomainLanz** [comment, marked as answer]
Hey @samueldrai! 👋🏻 

It seems you missed setting the `NODE_ENV` value to `production`.

```sh
NODE_ENV=production node bin/server.js
```

**Comment by samueldrai** [comment]
Thanks for the answer @RomainLanz !
